x-pack/filebeat/input/awss3/sqs_s3_event.go

```diff
@@ -211,19 +226,6 @@ func (p *sqsS3EventProcessor) keepalive(ctx context.Context, log *logp.Logger, w
 				"expires_at", time.Now().UTC().Add(p.sqsVisibilityTimeout))
 			p.metrics.sqsVisibilityTimeoutExtensionsTotal.Inc()
 
-			// Renew visibility.
```

Comment:
should we keep the renew part here to change the visibility timeout?

Reply:
This whole thing is wrapped in a for loop, so once the timer expires it'll call the same renewal code above, the only difference should be that this also calls it before the first select. (Does that answer your question? I'm not sure I followed what you meant.)

Reply:
oh yes...!!! Missed the for loop!! Thanks!!